spec/takeaway_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'takeaway'
+
+describe TakeAway do
+  let(:takeaway) { TakeAway.new }
+  let(:menu) {{"Vanilla Milkshake" => 3.99}}
+  let(:print_menu) {{"Vanilla Milkshake" => 3.99}}
+
+  ITEM = "Vanilla Milkshake"
+
+  it 'should have an empty basket' do
+    expect(takeaway.basket).to be_empty
+  end
+
+  it "shows the menu with the dishes and prices" do
+    expect(takeaway.print_menu)
```

Comment:
same here. You need to follow up `expect` with `.to` and an Rspec matcher to make an assertion about the output.